sdks/java/io/azure-cosmosdb/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id("org.apache.beam.module")
+  id("scala")
+}
+
+ext {
+  junitVersion = "5.9.1"
+  cosmosVersion = "4.37.1"
+  cosmosContainerVersion = "1.17.5"
+  //lombokVersion = "1.18.24"
+  bsonMongoVersion = "4.7.2"
+  log4jVersion = "2.19.0"
+}
+
+applyJavaNature(automaticModuleName: "org.apache.beam.sdk.io.azure.cosmosdb")
+
+description = "Apache Beam :: SDKs :: Java :: IO :: Azure Cosmos DB"
+ext.summary = "IO library to read and write Azure Cosmos DB services from Beam."
+
+dependencies {
+  implementation("org.scala-lang:scala-library:2.13.10")
+  implementation("com.azure:azure-cosmos:${cosmosVersion}")
+  implementation library.java.commons_io
+  permitUnusedDeclared library.java.commons_io // BEAM-11761
+  implementation library.java.slf4j_api
+  //implementation library.java.vendored_guava_26_0_jre
+  implementation project(path: ":sdks:java:core", configuration: "shadow")
+  //implementation library.java.jackson_annotations
+  //implementation library.java.jackson_core
+  //implementation library.java.jackson_databind
+  implementation("org.mongodb:bson:${bsonMongoVersion}")
+}
+//implementation("com.azure:azure-identity:1.0.8")
+//implementation("com.azure:azure-core:1.9.0")
+
+// LAMBOK
+dependencies {
+  //compileOnly("org.projectlombok:lombok:${lombokVersion}")
+  //annotationProcessor("org.projectlombok:lombok:${lombokVersion}")
+  //testCompileOnly("org.projectlombok:lombok:${lombokVersion}")
+  //testAnnotationProcessor("org.projectlombok:lombok:${lombokVersion}")
+}
+
+// TEST
+dependencies {
+  testImplementation("org.testcontainers:azure:${cosmosContainerVersion}")
+  testImplementation("com.outr:scribe_2.13:3.10.3")
+  testImplementation project(path: ":sdks:java:core", configuration: "shadowTest")
+  testImplementation library.java.mockito_core
+  testImplementation("org.junit.vintage:junit-vintage-engine:${junitVersion}")
+  testRuntimeOnly("org.apache.logging.log4j:log4j-api:$log4jVersion")
```

Comment:
Why do you need `log4j-api` during `testRuntimeOnly`? I would expect the API to be a `testImplementation` dependency and the backend for logging to be the runtime dependency. Does the cosmoDB client use it and not declare its deps?

Reply:
dependen only the slf4j, but I want to see the logs in the tests.

Reply:
Remember that you can use log4j or logback or any other